front/migrations/backfill_workspace_has_domains.ts

```diff
@@ -26,6 +26,7 @@ makeScript({}, async ({ execute }) => {
         await WorkspaceHasDomain.create({
           domain: workspace.allowedDomain,
           domainAutoJoinEnabled: true,
+          workspaceId: workspace.id,
```

Comment:
Weirdly Sequelize type is broken here and does not capture the required foreign key constraint.